src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/ZoneTagHelper.cs

```diff
@@ -41,6 +44,14 @@ public async override Task ProcessAsync(TagHelperContext context, TagHelperOutpu
             {
                 await shape.AddAsync(childContent, Position);
             }
+            else
+            {
+                _logger.LogWarning(
+                    "Unable to add shape to the zone using the <zone2> tag helper because the zone's type is " +
```

Comment:
`<zone>` instead of `<zone2>`

Reply:
@jtkech should we take a step further and actually allow shapes in position wrapper for rendering?